test/plugin_functional/test_suites/panel_actions/index.js

```diff
@@ -30,11 +30,12 @@
 
 import path from 'path';
 
-export const OPENSEARCH_DASHBOARDS_ARCHIVE_PATH = path.resolve(
```

Comment:
i think this was potentially exported because there might have been x-pack plugins that referenced it but i dont see none in the opensearch-project using.

Reply:
Hmm interesting. I was unable to add a new `ciGroup` until i removed these exports.